colibri/src/main.rs

```diff
@@ -58,17 +63,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         )
         .with_state(state);
 
-    let cors_patterns: Vec<Pattern> = args.api_cors
+    let cors_patterns: Vec<Pattern> = args
+        .api_cors
         .iter()
-        .filter_map(|pattern| {
-            match Pattern::new(pattern) {
-                Ok(glob) => {
-                    Some(glob)
-                },
-                Err(e) => {
-                    error!("Found pattern {} not valid for API CORS due to {}", pattern, e);
-                    None
-                }
+        .filter_map(|pattern| match Pattern::new(pattern) {
+            Ok(glob) => Some(glob),
+            Err(e) => {
+                error!(
+                    "Found pattern {} not valid for API CORS due to {}",
+                    pattern, e
+                );
+                None
             }
```

Comment:
why changes in the style here?

Reply:
most likely an issue from my ide. reverting it.